include/ntcore_c.h

```diff
@@ -305,6 +305,8 @@ int NT_NotifierDestroyed();
 /*
  * Remote Procedure Call Functions
  */
+ 
+#define TIMEOUT_INDEFINITE -1
```

Comment:
This needs an NT prefix.  Is this really necessary given the non-timeout variants?